webapp/app/static/jsx/components/AccordionOfResults.jsx

```diff
@@ -11,10 +11,10 @@ const Spinner = (<div id="spinner" className="justify-content-center">
 
```

Comment:
![image](https://user-images.githubusercontent.com/44105002/149240344-4987b8d7-dd70-4812-9c9d-59ccf10ee294.png)
it looks like the accordionOfResults is mobile-responsive until you expand one of the items. could you fix this?